docs/rules/newline-per-chained-call.md

```diff
@@ -23,12 +23,29 @@ d3
         42
     ])
     .enter()
-    .append('p')
+    .append("p")
     .text(function (d) {
         return "I'm number " + d + "!";
     });
 ```
 
+Another argument in favor of this style is that it improves the readability of diffs when something in the method chain is changed:
```

Comment:
Can you use "clarity" instead of "readability"? Minor nitpick, but readability is a loaded term that I've generally tried to avoid using in our docs(readability is personal).